specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -284,6 +367,14 @@
             "description": "Type: string (or Expression with resultType string)."
           },
           "description": "Preserve Rules."
+        },
+        "preserve": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "description": "Type: string (or Expression with resultType string)."
+          },
+          "description": "Preserve."
```

Comment:
This is not a good description

Reply:
I asked the owner and he said that this modification is feasible